feat: new create from image visuals

Kuchenpirat opened the pull request:
## What this PR does / why we need it:

New Visuals for the create from image functionality as the old visuals were a bit broken after the nuxt 3 upgrade and did not handle multiple images upload very gracefully. 

## Which issue(s) this PR fixes:

None, just didn't like the visuals too much, especially with multiple images adding up one below the other. 

## Special notes for your reviewer:

- merge after #5590 

## Testing

Local

## Screenshot
![image](https://github.com/user-attachments/assets/9c0c1435-7295-4b71-a63d-8035d6a9cd26)

Kuchenpirat committed:
upgade create from image visuals

Kuchenpirat committed:
Merge branch 'mealie-next' into upgrade-create-from-image-visuals

Kuchenpirat committed:
don't close menu when clicking

Kuchenpirat committed:
Merge remote-tracking branch 'upstream/mealie-next' into upgrade-create-from-image-visuals

michael-genson reviewed (changes requested):
Looks SOOO much better! A couple things I noticed:
1. It's possible to delete/edit the photos after submitting them to the backend. Can we disable those buttons after submitting?
2. The save button doesn't seem to be working after cropping/rotating

Kuchenpirat committed:
disable buttons on submission

Kuchenpirat committed:
fix image save

Kuchenpirat committed:
move save button out of menu

Kuchenpirat commented:
fixed the issues you found. 
I also moved the save button out of the menu and made it only visible if the image was changed.

![image](https://github.com/user-attachments/assets/8176dfd8-0c29-49e2-8e65-5c02393e72c0)

Kuchenpirat committed:
center menu and give it a bit of margin to the button

michael-genson reviewed (approved):
Love it!
